src/tests/validity/function_optional_argument_array_default.ab

```diff
@@ -0,0 +1,12 @@
+// Output
+// 1000
+
+fun sum_array(a : [Num] = [100,200,300,400]): Num {
+    let sum = 0
+    loop n in a {
+        sum += n;
+    }
+    return sum;
+}
+let result = sum_array();
+echo "{1000}"
```

Comment:
Did you mean to echo a hard coded value?  I suggest:
```
let result = sum_array()
echo "{result}"
```

Reply:
You changed `echo "{1000}"` to `echo 1000`; this still doesn't test the behaviour of `sum_array()`.  Can you change it to `echo result` like the other tests?